book/src/api-reference/jsonrpc-api.md

```diff
@@ -78,6 +78,33 @@ Requests can be sent in batches by sending an array of JSON-RPC request objects
 * Signature: An Ed25519 signature of a chunk of data.
 * Transaction: A Solana instruction signed by a client key-pair.
 
+## Configuring State Confidence
+
+Solana nodes choose which bank state to query based on confidence requirements
+set by the client. Clients may specify two parameters:
+* confirmations - how many confirmed blocks the node should require, up to `MAX_LOCKOUT_HISTORY`
+* percentage - how much of the cluster stake must have registered `confirmations`, as a decimal between 0.0 and 1.0
+
+The confidence parameter should be included as the last element in the `params` array:
+
+```bash
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "method":"getBalance", "params":["83astBRguLMdt2h5U1Tpdq5tjFoJ6noeGwaY3mDLVcri",{"confirmations":15,"percentage":0.60}]}' 192.168.1.88:8899
+```
+
+For convenience, the confirmations parameter can be set to `max`, which defaults
+to `MAX_LOCKOUT_HISTORY`
+
+```bash
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "method":"getBalance", "params":["83astBRguLMdt2h5U1Tpdq5tjFoJ6noeGwaY3mDLVcri",{"confirmations":"max"}]}' 192.168.1.88:8899
+```
+
+#### Defaults:
+Used if confidence configuration is not provided, or for missing parameters
+* `confirmations`: MAX_LOCKOUT_HISTORY
+* `percentage`: DEFAULT_PERCENT_CONFIDENCE (2/3 of the total cluster stake + 1)
```

Comment:
@CriesofCarrots Very thanks for 23c2a3f39cce0d417e04fd02ca4ce816eae92a35! Then this explanation just be `(2/3 of the total cluster stake)` or just empty? What does `+ 1` mean? `more than 2/3`? Maybe I think this needs a bit clarification.